src/hubot-untappd-friends.coffee

```diff
@@ -25,7 +25,7 @@
 # Author:
 #  sethington, stephenyeargin
 
-count_to_return = process.env.UNTAPPD_MAX_COUNT || 10
+count_to_return = process.env.UNTAPPD_MAX_COUNT || 5
```

Comment:
Just a preference here?

Reply:
With Slack formatting, the default of 10 fills the screen.  I thought about setting the `count_to_return` based on the adaptor in use, but that seemed needlessly complicated.  It's a user configurable option, so I'd be happy to return this to 10 if you prefer.

Reply:
:cool:, Nah, 5 is fine. I think we were wanting it slightly wider for IRC (lots of 🍻), but as you said, it's user configurable. I can add it to the release notes that the default behavior changed.